Cartfile

```diff
@@ -1,4 +1,4 @@
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 23.2.1
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 123.0
-github "mapbox/mapbox-directions-swift" ~> 2.8
+github "mapbox/mapbox-directions-swift" "maxim/NAVIOS-328-add-support-for-sapa-facilities"
```

Comment:
Will be updated after landing https://github.com/mapbox/mapbox-directions-swift/pull/780.